config/api.yml

```diff
@@ -117,6 +117,17 @@
     :klass: ChargebackRate
     :subcollections:
     - :rates
+  :container_deployments:
+    :description: Container Provider Deployment
+    :identifier: container_deployment
+    :klass: ContainerDeployment
+    :methods: *70174834084700
+    :options:
+    - :collection
```

Comment:
@alongoldboim can you point me to failing example and any error messages you got?